test/FsAutoComplete.Tests.Lsp/CodeFixTests.fs

```diff
@@ -1263,6 +1263,987 @@ let private generateUnionCasesTests state =
         """
   ])
 
+let private implementInterfaceTests state =
```

Comment:
This file is getting large enough that I almost want to start splitting the individual codefix tests into sub-files for organization purposes :) You shouldn't do this as part of this PR, just wanted to let you know I was having some mixed feelings about the length now.

Reply:
You're right: most likely too large. 
Navigating by sight (or scrollbar...) is now quite tricky (especially for testList with lots of cases (like ImplementInterface))

Tests for next CodeFix (GenerateUnionCases) is probably similar large  
-> better do it now than later  

-> I extracted some tests into their own files (only the biggest ones -- rest is still in one file -> can be extracted if someone changes or adds tests to them)

Reply:
you overachiever! thanks for that :)